src/models/tests/distance.test.ts

```diff
@@ -1,13 +1,15 @@
+/* eslint-disable prefer-destructuring,@typescript-eslint/dot-notation */
```

Comment:
Disabled for this file because we access some private members through array access for testing purposes, and the above would break the build.